src_new/util.idhub.js

```diff
@@ -261,11 +262,17 @@ exports.getUserIdParams = function (params) {
 			refThis.logWarning(CONSTANTS.MESSAGES.IDENTITY.M3, ex);
 		}
 	}
+	var ssoTimeout = window.PWT && window.PWT.ssoEnabled ? CONSTANTS.CONFIG.SSO_INTEGRATION_TIMEOUT : 0;
+	ssoTimeout += window.PWT.fbTimeout ? window.PWT.fbTimeout : 0;
 	if (userIdParams && userIdParams.params && userIdParams.params["loadATS"] == "true") {
-		refThis.initLiveRampAts(userIdParams);
+		setTimeout(function() {
+			refThis.initLiveRampAts(userIdParams); 
+		}, ssoTimeout);
 	}
 	if(userIdParams && userIdParams.params && userIdParams.params['loadIDP'] == 'true'){
-		refThis.initZeoTapJs(userIdParams);
+		setTimeout(function() {
```

Comment:
Please remove timeout as this sso feature is not going in this release.

Reply:
done